### PR TITLE
Changed default 'execute' behavior to False.

### PR DESCRIPTION
**Pull request by psigen**
Currently, it is very scary that calling a Plan...() function anywhere in the code without 'execute=False' can cause a robot to start moving and doing things.

It does not seem at all reasonable that this is the default behavior, especially now that we are increasingly using pipelined planning calls and post processing trajectories heavily before execution in demos-- where execute=True is exactly the wrong thing to do.

We should not have to pass a non-default argument to every single planning call to keep the robot from moving.

### CODE REVIEW

**Comment by psigen** [comment]
Hoping @cdellin and @jeking04 can back me up on this.

**Comment by jeking04** [comment]
I agree that it is probably the right thing to do to have ```execute=False``` be the default behavior, as I would not expect a call to Plan...() to do anything more than plan.

In practice the change might be a bit of a pain because we have to upgrade all current calls where we expect execution to pass this flag.  On the positive side, in much of our current demo we are already passing ```execute=False``` so its probably not that much code that will need changed. Maybe an action in the action library, and some older demos that are probably broken in much bigger ways anyway.

In summary, I agree with this change.

**Comment by mkoval** [comment]
I'm fine with this change. I'm also a bit concerned about the amount of code we'll have to update if we make this change.

**Comment by psigen** [comment]
Can I merge this then?  Does someone need to update some code somewhere?

**Comment by mkoval** [comment]
Here comes the breakage...
